### ISSUE TITLE
1.3.0 fails with docker

### ISSUE THREAD

**Issue by eygraber** [danger/kotlin#270]
```
Pull down action image 'ghcr.io/danger/danger-kotlin:1.3.0'
  /usr/bin/docker pull ghcr.io/danger/danger-kotlin:1.3.0
  Error response from daemon: manifest unknown
  Warning: Docker pull failed with exit code 1, back off 1.033 seconds before retry.
  /usr/bin/docker pull ghcr.io/danger/danger-kotlin:1.3.0
  Error response from daemon: manifest unknown
  Warning: Docker pull failed with exit code 1, back off 8.75[2](https://github.com/eygraber/immich-kmp/actions/runs/8382266073/job/22955706091?pr=21#step:2:2) seconds before retry.
  /usr/bin/docker pull ghcr.io/danger/danger-kotlin:1.3.0
  Error response from daemon: manifest unknown
Error: Docker pull failed with exit code 1
```

**Comment by Vacxe** [comment]
@eygraber if this issue related to GitHub Actions please take a look on documentation

I guess you using example from this repository flows, like

```
      - name: Danger
        uses: danger/kotlin@master
        with:
          args: "--id DangerKotlinAction"
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```

At this moment I migrating to new docker and github action implementation 

As Documentation says https://github.com/danger/kotlin?tab=readme-ov-file#github-actions you should use versioned action,  and that make sense for stability in case if master branch have broken backward compatibility

So, just replace `uses: danger/kotlin@master` with `uses: danger/kotlin@1.2.0` 

Cheers, let me know if it help

**Comment by eygraber** [comment]
I'm using it like this:

```
      - name: Danger
        uses: docker://ghcr.io/danger/danger-kotlin:1.3.0
        with:
          args: --failOnErrors --no-publish-check
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by Vacxe** [comment]
@eygraber even better. Why did you decide to use 1.3.0 if it still in "Pre-Release"? 
As well, it always make sense to verify if docker package even exist https://github.com/orgs/danger/packages/container/package/danger-kotlin

My suggestion for now, rollback to 1.2.0 and use it until new release will be published. As well, docker calls will be slightly different  with 1.3.0.

**Comment by Vacxe** [comment]
@eygraber does it help you? Can we close the issue?

**Comment by eygraber** [comment]
Yes, I didn't realize it was a pre-release? I just got the email from GitHub about the release and tried updating.

**Comment by Vacxe** [comment]
@eygraber just in case I've published 1.3.0 release with new docker. But the way how it using now is slightly different. All refrected in new documentation (Main Readme). That not the best way to have `Entry point` in lib dockers because it create some complexity it extending.

With 1.3.0 should be 

```yaml
- name: Danger
  uses: docker://ghcr.io/danger/danger-kotlin:1.3.0
  run: danger-kotlin --failOnErrors --no-publish-check
  env:
    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Cheers, K

**Comment by eygraber** [comment]
GitHub Actions doesn't allow `use` and `run` in the same step. It fails the whole workflow.
